src/commands/rage.ts

```diff
@@ -0,0 +1,63 @@
+import supportsColor from 'supports-color';
+import { getBorderCharacters, table } from 'table';
+
+import BaseCommand from '../lib/baseCommand.js';
+import configstore from '../lib/configstore.js';
+import getCurrentConfig from '../lib/getCurrentConfig.js';
+import isCI, { ciName, isGHA } from '../lib/isCI.js';
+
+export default class RageCommand extends BaseCommand<typeof RageCommand> {
```

Comment:
Haven't written any unit tests for this but will if folks are into the command idea.